fedimint-api/src/config.rs

```diff
@@ -71,8 +72,10 @@ impl FederationId {
         Self(threshold_crypto::PublicKey::from(G1Projective::identity()))
     }
 
-    fn from_bytes(bytes: [u8; 48]) -> Self {
-        Self(threshold_crypto::PublicKey::from_bytes(bytes).expect("Invalid pub-key"))
+    fn from_bytes(bytes: [u8; 48]) -> Result<Self> {
+        threshold_crypto::PublicKey::from_bytes(bytes)
+            .map_err(anyhow::Error::from)
```

Comment:
This and one above `.map_err(anyhow::Error::from)` seems redundant I think. `?` does a `From` conversion.